distage/distage-framework-docker/src/main/scala/izumi/distage/docker/examples/CassandraDocker.scala

```diff
@@ -0,0 +1,27 @@
+package izumi.distage.docker.examples
+
+import izumi.distage.docker.ContainerDef
+import izumi.distage.docker.Docker.{ContainerConfig, DockerPort}
+import izumi.distage.model.definition.ModuleDef
+import izumi.fundamentals.reflection.Tags.TagK
+
+object CassandraDocker extends ContainerDef {
+  val primaryPort: DockerPort = DockerPort.TCP(9042)
+
+  override def config: Config = {
+    ContainerConfig(
+      image = "cassandra:3.0.16",
```

Comment:
Seems like the latest version is `3.11.6` ? https://hub.docker.com/_/cassandra/ Since these are just bundled examples, we may as well ship the latest versions, if someone needs an older one they can make their own copy